lib/common/strings.c

```diff
@@ -687,50 +687,46 @@ pcmk__str_table_dup(GHashTable *old_table)
  * \internal
  * \brief Add a word to a string list of words
  *
- * \param[in,out] list       Pointer to current string list (may not be NULL)
- * \param[in,out] len        If not NULL, must be set to length of \p list,
- *                           and will be updated to new length of \p list
+ * \param[in,out] list       Pointer to current string list (may not be \p NULL)
+ * \param[in]     init_size  \p list will be initialized to this size, if it
+ *                           needs initialization
```

Comment:
It's worth making it more clear that `init_size` can be 0.

Reply:
Fair point, I will